lib/messenger.rb

```diff
@@ -0,0 +1,30 @@
+# Understands the delivery of confirmation message.
+
+require 'rubygems' # not necessary with ruby 1.9 but included for completeness
```

Comment:
if it's not needed in the ruby version you are using, don't leave it in your code.